src/testing/AddTestForm.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Box, Text, Label, Input, Checkbox, Button } from 'theme-ui';
+import { useFormik } from 'formik';
+
+import { CreateTestCommand, TestType, FilledSchema, ObjectSchema, FieldSchema } from '../api';
+
+const AnyBox = Box as any;
+
+export const AddTestForm = ({
+  testType,
+  onComplete,
+}: {
+  testType: TestType;
+  onComplete: (command: CreateTestCommand) => any;
+}) => {
+  // TODO: validation
+  // TODO: supporting more arbitrary schemas
+  const form = useFormik({
+    initialValues: getInitialValues(testType.resultsSchema),
+    onSubmit(details: FilledSchema) {
+      onComplete({ testTypeId: testType.id, results: { details } });
+    },
+  });
+
+  return (
+    <>
+      <Text>Result</Text>
```

Comment:
You could use the schema title here if there's one present.

Reply:
It looks a bit ridiculous - the select is on the top with the title, under which you have the same exact title